src/huggingface_hub/hf_api.py

```diff
@@ -170,6 +171,37 @@ class BlobLfsInfo(TypedDict, total=False):
     sha256: str
 
 
+@dataclass
+class CommitInfo:
+    """Data structure containing information about a newly created commit.
+
+    Returned by [`create_commit`].
+
+    Args:
+        commit_url (`str`):
+            Url where to find the commit.
+
+        commit_message (`str`):
+            The summary (first line) of the commit that has been created.
+
+        commit_description (`str`):
+            Description of the commit that has been created. Can be empty.
+
+        oid (`str`):
+            Commit hash id. Example: `"91c54ad1727ee830252e457677f467be0bfd8a57"`.
+
+        pr_url (`str`, *optional*):
+            Url to the PR that has been created, if any. Populated when `create_pr=True`
+            is passed.
+    """
+
+    commit_url: str
+    commit_message: str
+    commit_description: str
+    oid: str
+    pr_url: Optional[str] = None
```

Comment:
would pr_number also be useful (maybe not)

Reply:
yes why not. We have an internal helper for that but it would be easy to present it to end users.

Reply:
Added `pr_revision` and `pr_num` in https://github.com/huggingface/huggingface_hub/pull/1066/commits/f606c58a818310bf38eab0b1d4f1f4e39f705171